.github/workflows/tests.yml

```diff
@@ -29,54 +29,12 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ github.event.pull_request.head.sha }}
-      - name: Setup MacOS
-        if: matrix.os == 'macos-latest'
+      - name: Tests debug build
```

Comment:
We should be able to test 32-bit Windows and Linux here as well.

Reply:
I filed #531, #532 and #533.